### PR TITLE
Alert feature complete

### PR DESCRIPTION
**Pull request by Dark-Knight499**
updates alert.ts

### CODE REVIEW

File: src/discord/alert.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    const maxlevel = levels.find(level => max_rating >= level.minRating);
+    const currlevel = levels.find(level => curr_rating >= level.minRating)
+    if (maxlevel.minRating > currlevel.minRating) {
+        const msg = `🎉 Congratulations ${handle} ! You've reached a new level: ${currlevel.name} with a rating of ${curr_rating}! Keep up the great work! 🎉`;
+
+        if (channel.isTextBased() && channel.isSendable())
+            channel.send(msg);
+    }
+}
+alertNewLevel("Harsh",1200,1000)
```
Comment (keerthivasansa) [inline src/discord/alert.ts]: Can you remove this?

**Review by keerthivasansa — CHANGES_REQUESTED** [review]

File: src/discord/alert.ts (hunk 1 above)
Comment (Dark-Knight499) [reply src/discord/alert.ts]: Done. Please check.

**Comment by keerthivasansa** [comment]
Can you resolve the conflicts? or I can do them if you want.

**Comment by Dark-Knight499** [comment]
Resolved them

**Comment by keerthivasansa** [comment]
awesome

**Review by keerthivasansa — APPROVED** [review]

**Pull request merged**